pom.xml

```diff
@@ -294,7 +294,7 @@
                 <configuration>
                     <properties>
                         <inceptionYear>${project.inceptionYear}</inceptionYear>
-                        <year>2013</year>
+                        <year>2016</year>
```

Comment:
This is the current year, it must be changed. There is a maven plugin that checks that the copyright notice has dates that look like `<inceptionYear>-<year>`